io.github.tralph3.Steam_Metadata_Editor.metainfo.xml

```diff
@@ -0,0 +1,368 @@
+</p>
+<p>
+  Then what do we do exactly? We format all the modified data this way (the function <code>format_data</code> takes care of this in my program), and then pass the result through SHA-1, and we get the checksum. <em>However</em>, there is one final thing that made me scratch my head for months that I couldn&apos;t figure out alone. Some apps have backslashes in their data. These backslashes are usually used for directory paths and stuff like that, and as you may know, backslashes are escape characters.
+</p>
+<p>
+  So... what does this mean to us? It means that I spent months trying to figure out why I could produce valid checksums for some games, while others failed all the time. When generating the checksum, you need to add a second backslash to every single one that appears. That&apos;s why in <code>format_data</code> I use <code>replace</code> to change this: <code>\\</code> to this <code>\\\\</code>. As the backslash is an escape character, you need another backslash to escape its own special functionality, so there it is, that&apos;s the final piece of the puzzle to produce valid checksums.
+</p>
+<p>
+  ## Special Thanks
+</p>
+<p>
+  Although I explained it all somewhat gracefully, that doesn&apos;t mean I managed to discover it myself sadly. I connected the dots later, but the actual information was provided to me by <em>Tim Green</em>, creator of [Steam Edit](https://steamedit.tg-software.com/). My application intends to be a cross-platform and open-source alternative to it, but in no way do I intend to make it obsolete or discredit <em>Tim</em>, he has been a great help in the creation of this program, and I&apos;m positive I wouldn&apos;t have been able to create it without his help. A big thanks to him.
+</p>
+<p>
+  I&apos;d also like to thank [<em>Leonid</em>](https://github.com/leovp) for indirectly aiding me by making the source code of [steamfiles](https://github.com/leovp/steamfiles) freely available to anyone, it has greatly helped me understand how the file worked, and I even borrowed some code.
+</p>
+<p>
+  Finally, thanks to [<em>xPaw</em>](https://github.com/xPaw) for documenting an update to the file format that included an extra checksum in [this](https://github.com/SteamDatabase/SteamAppInfo) repo.
+</p>
+</description>
+
+<launchable type="desktop-id">steam-metadata-editor.desktop</launchable>
+<screenshots>
+  <screenshot type="default">
+    <image>https://github.com/tralph3/Steam-Metadata-Editor/blob/d9d62f791b91f21adcc3d0835eb3a148218a52b7/img/layout.png?raw=true</image>
+  </screenshot>
+  <screenshot>
+    <image>https://github.com/tralph3/Steam-Metadata-Editor/blob/d9d62f791b91f21adcc3d0835eb3a148218a52b7/img/edits2.png?raw=true</image>
+  </screenshot>
+  <screenshot>
+    <image>https://github.com/tralph3/Steam-Metadata-Editor/blob/d9d62f791b91f21adcc3d0835eb3a148218a52b7/img/mod.png?raw=true</image>
+  </screenshot>
+</screenshots>
```

Comment:
missing at least `url` and `content_rating`

Reply:
also missing `releases`

Reply:
I don't really use version numbers for this application. I tried adding <release>1.0</release> but appstream-glib still complains that the release tag is missing.

Reply:
that's not the syntax for release.

Reply:
So how? I looked here: https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/

There's no mention of the release tag, I can't find it.

Reply:
Never mind, found it.